Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,17 @@
+class Pacapt < Formula
+
+  homepage "https://github.com/icy/pacapt" 
+  url "https://github.com/icy/pacapt.git", :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4"
+  version "v2.0.2"
+
+  def install
+    system "make", "install", "BINDIR=#{bin}"
+  end
+
+  test do
+    system bin/"pacapt", "-V"
+    
+    assert_equal %x(brew list | wc -w), %x(pacapt -Q '.' | wc -w)
```

Comment:
It should be reliable since both should return just a list of names of installed packages. But it seems to be syntactically wrong anyway. I am new to ruby. ;)

At last I've used the
```ruby
system bin/"pacapt -Q '.'"
```
as you recommanded ;)